src/main/scala/Url.scala

```diff
@@ -1,3 +1,98 @@
-object Url {
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.model.{ContentTypes, HttpEntity, StatusCodes}
+import akka.http.scaladsl.server.Directives._
+import akka.util.Timeout
+import cats.effect.IO
 
+import scala.concurrent.duration._
+import java.sql.Timestamp
+import scala.language.postfixOps
+import spray.json._
+import doobie._
+import doobie.implicits._
+import doobie.implicits.javasql._
+
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+case class UrlIn(userId: Int, value: String, timestamp: String)
+case class UrlOut(userId: Int, value: String, timestamp: Timestamp)
+
+object DatabaseConnection {
+  implicit val cs = IO.contextShift(ExecutionContext.global)
+
+  val xa = Transactor.fromDriverManager[IO](
+    "org.postgresql.Driver", "jdbc:postgresql:browser_history", "postgres", ""
+  )
+
+  def createUrl(userId: Int, timestamp: String, value: String) = {
```

Comment:
I suggest to name it `insertUrl`/`storeUrl`, since the `creation` action is more related to the creation of the object.

Also, what's the returned type of this method?

Reply:
The returned type is Int. If an update is executed successfully it returns 1

Reply:
Maybe it would help to specify the return type. :) It would help when a new person comes and needs to quickly call the methods, without extensive knowledge of the actual implementation.